resources/js/game_list_element.js

```diff
@@ -524,7 +524,7 @@ export default class GameListElement extends React.Component {
 					}}
 					key={itemKey++}
 				>
-					Share
+					Invite
```

Comment:
Invite seems to make people think it will send something somewhere. I feel that share is truer, since it it actually only copies a URL that leads to the game. WDYT?